includes/notes/class-wc-payments-notes-additional-payment-methods.php

```diff
@@ -31,7 +31,15 @@ class WC_Payments_Notes_Additional_Payment_Methods {
 	 */
 	public static function get_note() {
 		// Show this notice only if UPE settings preview is disabled, and UPE flag is not enabled.
-		if ( ! WC_Payments_Features::is_upe_settings_preview_enabled() || WC_Payments_Features::is_upe_enabled() ) {
```

Comment:
Don't like `||`? :)

Reply:
I thought it was going to get a little long with 3 `||`s 😬